include/ck/tensor_operation/gpu/element/unary_element_wise_operation.hpp

```diff
@@ -412,14 +424,14 @@ struct Swish
 {
     Swish(float beta = 1.0f) : beta_(beta) {}
 
-    template <typename T>
-    __host__ __device__ void operator()(T& y, const T& x) const
+    template <typename Y, typename X>
+    __host__ __device__ void operator()(Y& y, const X& x) const
     {
-        static_assert(is_same<T, float>::value || is_same<T, double>::value ||
-                          is_same<T, ck::half_t>::value,
+        static_assert(is_same<X, float>::value || is_same<X, double>::value ||
+                          is_same<X, ck::half_t>::value,
                       "Data type is not supported by this operation!");
 
-        y = x / (ck::type_convert<T>(1) + ck::math::exp(-beta_ * x));
+        y = x / (ck::type_convert<Y>(1) + ck::math::exp(-beta_ * x));
```

Comment:
I did not did well before.
`float bx = -beta_ * type_convert<float>(x)`
` y = type_convert<Y>(x / (ck::type_convert<float>(1) + ck::math::exp(bx)););`